cmd/web/main.go

```diff
@@ -75,6 +75,10 @@ func main() {
 	api.DELETE("/clusters", server.ClusterDeleteHandler)
 	api.GET("/resourcegroups/:accountID", server.ResourceGroupHandler)
 	api.GET("/iam/accessGroups/:accountID", server.AccessGroupsHandler)
+	api.PUT("/groups/:accessGroupID/members", server.AddMemberHandler)
+	api.POST("/accounts/:accountID/users", server.InviteUserHandler)
```

Comment:
should the top level resource be account or user?

Reply:
```suggestion
      api.POST("/users/account/:accountID", server.InviteUserHandler)
```

Reply:
This was the format that the IBM Cloud api follows so I was keeping it close to that. To me it makes sense because these user management actions are done in the context of an account (invited to account, removed from account). If you prefer having user as the top level resource, I can make the change.